src/plain-animator.ts

```diff
@@ -90,9 +94,9 @@ export class PlainAnimator {
 
   protected updateFrame() {
     const tileHeight = 1 / this.tilesAmountVertically;
-
-    const currentColumn = this.currentFrame % this.tilesAmountHorizontally;
-    const currentRow = Math.floor(this.currentFrame / this.tilesAmountHorizontally);
+    const currentFrameWithOffset = this.currentFrame + this.currentFrameOffset
```

Comment:
Hey, thanks for the pull request! 😍 

I have one question tho, how this exact line works, I mean if I set `currentFrameOffset` to 10, then on init
```
this.currentFrame = 10
```
and in `updateFrame` method:
```
const currentFrameWithOffset = 10 + 10 = 20
```

is this really what we expect? I mean it looks like we are starting with frame number 10 and then we are moving in the next step to frame number 20.

Its highly possible that I'm missing something, so please explain your thought on that. I'm really happy to merge that as soon as we agreed on that :)